database/sqltypes/sqltypes.go

```diff
@@ -70,6 +70,11 @@ func MakeString(b []byte) Value {
 	return Value{String{b, false}}
 }
 
+// MakeUtf8String makes a String value from a []byte.
+func MakeUtf8String(s string) Value {
```

Comment:
I actually need them and I'm adding this because there is no way to create it. I have some code (not yet committed) that basically marshalls the sql types and stores them on protobuffs. Then, later, I might need to shove them back in SQL. So, I need to re-create them. So, If I got a StringValue with uff set to true, I need to create the same type. This is the only one that I couldn't find a factory for. The others are fine.